tests/Test.php

```diff
@@ -70,6 +70,8 @@ public function testAddHeaderData() {
 	 * Test the edge_integrations_enabled function.
 	 */
 	public function testEIEnabled() {
+		remove_all_filters( 'pantheon.ei.parsed_geo_data' );
+		remove_all_filters( 'pantheon.ei.parsed_interest_data' );
```

Comment:
This is somewhat related to my question [above](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/pull/94/files#r847739833).

Reply:
Yep, both of those are there to ensure that when testing, we're not inheriting filtered values from other places (since we're not using a real WPUnit environment that clears out the instance in between test runs).

I defer to my initial statement that if you're filtering that stuff, you're modifying the data intentionally, so ymmv, but if there's a place in the documentation that we could be more explicit about the side effects, I'd be happy to update.